pkg/obfuscator/mac_address.go

```diff
@@ -33,12 +33,12 @@ func (m *macAddressObfuscator) Contents(s string) string {
 		replacement := m.obfsGenerator.generateReplacement(match, m.ReplacementTracker)
 		s = strings.ReplaceAll(s, mac, replacement)
 		// also add the original (non-cleaned) string, this is only used for human review in the final report
-		m.ReplacementTracker.AddReplacement(mac, replacement)
+		m.ReplacementTracker.AddReplacement(mac, Replacement{ReplacedWith: replacement})
 	}
 	return s
 }
 
-func (m *macAddressObfuscator) Report() map[string]string {
+func (m *macAddressObfuscator) Report() map[string]Replacement {
```

Comment:
I think we don't even need to define the method here, since the ReplacementTracker is an embedded struct -> candidate for removal